src/sb_pointcloud_processing/include/DBSCAN.h

```diff
@@ -78,6 +100,36 @@ class DBSCAN {
      */
     void findNeighbors();
 
+    /**
+     * Finds the neighbours in parallel
+     * Number of threads is defined by this->_num_threads
+     */
+    void findNeighborsParallel();
+
+    /**
+     * Finds the neighbours sequentially
+     */
+    void findNeighborsSequential();
```

Comment:
What was the reasoning behind keeping `findNeighboursParallel` and `findNeighboursSequential`? I would think they would be equivalent if we just set `num_threads` to 1?

Reply:
using openmp instead, not relevant anymore